app/src/main/res/layout/exo_player_control_view.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<merge xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!-- 0dp dimensions are used to prevent this view from influencing the size of
+         the parent view if it uses "wrap_content". It is expanded to occupy the
+         entirety of the parent in code, after the parent's size has been
+         determined. See: https://github.com/google/ExoPlayer/issues/8726.
+    -->
+    <!-- Change from media3: Make transparent -->
```

Comment:
Please don't make these completely transparent, it makes the controls impossible to see in some cases.

I just fixed something similar when viewing profiles.

Reply:
They are completely transparent in the current (Tusky 21) implementation. With our idiom of the controls appearing and disappearing at different moments, a full-screen curtain means a really unpleasant full-screen blinking effect… at minimum, I really want to find some way to avoid that.

Would it help if we used icons with internal backgrounds? For example a black play symbol inset in a white square.

(There's still a background on the play bar— just not the icons— and the play bar background is not unpleasant because it covers a small part of the screen and potentially doesn't cover the video at all.)

Reply:
> Would it help if we used icons with internal backgrounds? For example a black play symbol inset in a white square.

Yes, that, but it should probably be a circle, it should use the Tusky colours, and there can be some transparency.

Take a look at https://github.com/tuskyapp/Tusky/pull/3272 - the trick is to use a `LayerDrawable` to compose the background drawable and the drawable with the icon.